src/index.ts

```diff
@@ -51,7 +55,7 @@ export function createIPXHandler ({
   responseHeaders,
   localPrefix,
   ...opts
-}: IPXHandlerOptions = {}) {
+}: IPXHandlerOptions = {}, loadSourceImage = defaultLoadSourceImage) {
```

Comment:
Is this just to allow testing?

Reply:
Yeah, that was the main reason, but we were already importing `loadSourceImage`, so why not compose instead.